packages/apputils-extension/style/notification.css

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+@import '~react-toastify/dist/ReactToastify.min.css';
+
+:root {
+  --toastify-color-light: var(--jp-layout-color1);
+  --toastify-color-dark: var(--jp-layout-color1);
+  --toastify-color-info: var(--jp-info-color1);
+  --toastify-color-success: var(--jp-success-color1);
+  --toastify-color-warning: var(--jp-warn-color1);
+  --toastify-color-error: var(--jp-error-color1);
+  --toastify-color-transparent: rgba(255, 255, 255, 0.7);
+  --toastify-icon-color-info: var(--toastify-color-info);
+  --toastify-icon-color-success: var(--toastify-color-success);
+  --toastify-icon-color-warning: var(--toastify-color-warning);
+  --toastify-icon-color-error: var(--toastify-color-error);
+  --toastify-toast-width: 320px;
```

Comment:
Should sizes use adaptive measurements e.g. `em`, `rem` etc to make toasts responsive for different screen sizes and layouts?